README.md

```diff
@@ -12,7 +12,7 @@ The backend of the website is the core of the interactive functionality of the g
 [Submit an issue](https://github.com/momentum-mod/website/issues/new) with some pictures and/or web console error output!
 
 ## *Want to help out?*
-[Join our Discord](https://discord.gg/wQWkRb6) and let us know! We also accept pull requests, *but make sure the issue/feature isn't already assigned to someone!*
+[Join our Discord](https://discord.gg/momentummod) and let us know! We also accept pull requests, *but make sure the issue/feature isn't already assigned to someone!*
 
 ## Looking for our documentation site?
```

Comment:
While we're here can we fix this header?

```suggestion
## *Looking for our documentation site?*
```

Reply:
Done